### ISSUE TITLE
Error: Cannot read property 'split' of undefined

### ISSUE THREAD

**Issue by Radiergummi** [nuxt-community/google-fonts-module#12]
When I try to run nuxt after installing the module, the following error appears:
```
 FATAL  Cannot read property 'split' of undefined
  at node_modules/google-fonts-helper/dist/index.js:181:31
  at Array.forEach (<anonymous>)
  at node_modules/google-fonts-helper/dist/index.js:180:23
  at Array.forEach (<anonymous>)
  at convertFamiliesObject (node_modules/google-fonts-helper/dist/index.js:157:12)
  at Function.parse (node_modules/google-fonts-helper/dist/index.js:414:22)
  at node_modules/@nuxtjs/google-fonts/lib/module.js:35:38
  at Array.map (<anonymous>)
  at node_modules/@nuxtjs/google-fonts/lib/module.js:35:8
  at node_modules/hable/dist/hable.js:1:1052
  at node_modules/hable/dist/hable.js:1:270
  at processTicksAndRejections (internal/process/task_queues.js:93:5)
  at async Nuxt.callHook (node_modules/hable/dist/hable.js:1:1021)
  at async Builder.build (node_modules/@nuxt/builder/dist/builder.js:5605:5)
  at async Object._buildDev (node_modules/@nuxt/cli/dist/cli-dev.js:106:5)
  at async Object.startDev (node_modules/@nuxt/cli/dist/cli-dev.js:64:7)


   ╭──────────────────────────────────────────────────────────╮
   │                                                          │
   │   ✖ Nuxt Fatal Error                                     │
   │                                                          │
   │   TypeError: Cannot read property 'split' of undefined   │
   │                                                          │
   ╰──────────────────────────────────────────────────────────╯

error Command failed with exit code 1.
```

I have configured a custom font, but it doesn't matter whether I completely remove the configuration. I also dumped the value the where the error occurs, which yields `Roboto:100,300,400,500,700,900`.

Environment:
- Module version: `1.0.1`  
- Nuxt version: `2.14.4`

Configuration file:
```js
export default {
    buildModules: [
        '@nuxtjs/vuetify',
        '@nuxtjs/google-fonts',
    ],

    googleFonts: {
        families: {
            Poppins: [ 100, 300, 600 ],
        },
    },
};

```

**Comment by ricardogobbosouza** [comment]
Hi @Radiergummi 
I was unable to reproduce this problem.
You could try to reinstall this module to see if the problem persists

**Comment by gsqrt2** [comment]
Have the same problem on multiple installations

**Comment by ricardogobbosouza** [comment]
@Radiergummi, @gsqrt2 
Thanks for reporting

Fixed on v1.0.3

**Comment by Radiergummi** [comment]
@ricardogobbosouza Sorry for not responding, I was off duty for a few days.  
Thank you so much for fixing! 🎉
